.github/workflows/check_guest_id.yaml

```diff
@@ -0,0 +1,92 @@
+name: Check guest ID
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "release/**"
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  check-guest-id:
+    name: Check guest ID
+    runs-on: aws-linux-medium
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+          submodules: recursive
+
+      - name: Install contracts prerequisites
+        uses: ./.github/actions/contracts-prerequisites
+
+      - name: Build guest artifacts
+        uses: ./.github/actions/build-guest-artifacts
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: guest-artifacts
+          path: guest_artifacts
+
+      - name: Check guest ID on mainnet
+        shell: bash
+        run: |
```

Comment:
Discussed live - we discussed to try out an approach closer to current logic - using solidity scripts and bash files in `bash/verifiers_management`.

Reply:
done